src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,41 @@
+.assess {
+  margin-left: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 164px;
+  align-items: flex-end;
+}
+
+.review {
+  font-size: 10px;
+  line-height: 15px;
+}
+
+.stars {
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-position: 50%;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--4 {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
This styles can be applied to `stars` itself instead of modifier, as they should be the same